exercises/elm-2-a/.meta/Example.Dict.elm

```diff
@@ -0,0 +1,50 @@
+module AggregateScorers exposing (..)
```

Comment:
The second part of the exercise also requires storing the possible starting points, of which each student will randomly get assigned one.

As an example, Go has defined the following files in [go-2-a exercise](https://github.com/exercism/research_experiment_1/tree/master/exercises/go-2-a/.meta):

- `start_1.go`
- `start_1_example.go`
- `start_2.go`
- `start_2_example.go`
- `start_3.go`
- `start_3_example.go`

For this PR, this would merely mean copying the two example files from the `elm-1-a` exercise and copying them to the `.meta` directory of the `elm-2-a` directory, with their names something like:

- `Start.Dict.Elm`
- `Start.List.Elm`